images/win/scripts/Installers/Windows2019/Install-VS2019.ps1

```diff
@@ -2,114 +2,20 @@
-              '--add Microsoft.VisualStudio.Component.VC.14.25.x86.x64 ' + `
-              '--add Microsoft.VisualStudio.Component.Windows10SDK.16299 ' + `
-              '--add Microsoft.VisualStudio.Component.Windows10SDK.17134 ' + `
-              '--add Microsoft.VisualStudio.Component.Windows10SDK.17763 ' + `
-              '--add Microsoft.VisualStudio.Component.Windows10SDK.18362 ' + `
-              '--add Microsoft.VisualStudio.Component.Windows10SDK.19041 ' + `
-              '--add Microsoft.VisualStudio.Component.WinXP ' + `
-              '--add Microsoft.VisualStudio.Component.Workflow.BuildTools ' + `
-              '--add Microsoft.VisualStudio.ComponentGroup.Azure.CloudServices ' + `
-              '--add Microsoft.VisualStudio.ComponentGroup.Azure.ResourceManager.Tools ' + `
-              '--add Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Llvm.Clang ' + `
-              '--add Microsoft.VisualStudio.ComponentGroup.Web.CloudTools ' + `
-              '--add Microsoft.VisualStudio.Workload.Azure ' + `
-              '--add Microsoft.VisualStudio.Workload.Data ' + `
-              '--add Microsoft.VisualStudio.Workload.DataScience ' + `
-              '--add Microsoft.VisualStudio.Workload.ManagedDesktop ' + `
-              '--add Microsoft.VisualStudio.Workload.ManagedGame ' + `
-              '--add Microsoft.VisualStudio.Workload.NativeCrossPlat ' + `
-              '--add Microsoft.VisualStudio.Workload.NativeDesktop ' + `
-              '--add Microsoft.VisualStudio.Workload.NativeGame ' + `
-              '--add Microsoft.VisualStudio.Workload.NativeMobile ' + `
-              '--add Microsoft.VisualStudio.Workload.NetCoreTools ' + `
-              '--add Microsoft.VisualStudio.Workload.NetCrossPlat ' + `
-              '--add Microsoft.VisualStudio.Workload.NetWeb ' + `
-              '--add Microsoft.VisualStudio.Workload.Node ' + `
-              '--add Microsoft.VisualStudio.Workload.Office ' + `
-              '--add Microsoft.VisualStudio.Workload.Python ' + `
-              '--remove Component.CPython3.x64 ' + `
-              '--add Microsoft.VisualStudio.Workload.Universal ' + `
-              '--add Microsoft.VisualStudio.Workload.VisualStudioExtension ' + `
-              '--add Component.MDD.Linux ' + `
-              '--add Component.MDD.Linux.GCC.arm'
+$requiredComponents = Get-ToolsetContent | Select-Object -ExpandProperty visualStudio | Select-Object -ExpandProperty workloads
+$workLoads = @("--allWorkloads --includeRecommended")
+$workLoads +=  $requiredComponents | ForEach-Object { "--add $_" }
+$workLoads += "--remove Component.CPython3.x64"
+$workLoadsArgument = [String]::Join(" ", $workLoads)
 
-$ReleaseInPath = "Enterprise"
-$BootstrapperUrl = "https://aka.ms/vs/16/release/vs_${ReleaseInPath}.exe"
+$releaseInPath = Get-ToolsetContent | Select-Object -ExpandProperty visualStudio | Select-Object -ExpandProperty edition
+$bootstrapperUrl = "https://aka.ms/vs/15/release/vs_${releaseInPath}.exe"
```

Comment:
you are updating 16 -> 15 - is it correct?

Reply:
Thank you, changed back to 16.